.storybook/preview-head.html

```diff
@@ -1,3 +1,7 @@
 <script>
   window.wp = {};
 </script>
+
+<!-- REMOVE ONCE jQuery is removed from the libs. -->
```

Comment:
Do we have an issue for this? Because _TODO_s are often forgotten in the code.

Reply:
yes, there is an open issue for this:
https://github.com/infinum/eightshift-frontend-libs/issues/42#event-2893662111

Reply:
Ok, then this comment needs to be removed 🙂